src/uu/stat/src/stat.rs

```diff
@@ -25,25 +26,23 @@ use std::os::unix::fs::{FileTypeExt, MetadataExt};
 use std::path::Path;
 use std::{env, fs};
 
-use std::collections::HashMap;
 use thiserror::Error;
-use uucore::locale::{get_message, get_message_with_args};
 
 #[derive(Debug, Error)]
 enum StatError {
-    #[error("{}", get_message_with_args("stat-error-invalid-quoting-style", HashMap::from([("style".to_string(), style.clone())])))]
+    #[error("{}", translate!("stat-error-invalid-quoting-style", "style" => style.clone()))]
     InvalidQuotingStyle { style: String },
-    #[error("{}", get_message("stat-error-missing-operand"))]
+    #[error("{}", translate!("stat-error-missing-operand", "util_name" => uucore::execution_phrase()))]
```

Comment:
Is this intentional?
```suggestion
    #[error("{}", translate!("stat-error-missing-operand"))]
```